CHANGES.md

```diff
@@ -1,5 +1,23 @@
 # Changes
 
+## edge-23.6.1
+
+This edge release changes the behavior of the CNI plugin to run exclusively in
+"chained mode". Instead of creating its own configuration file, the CNI plugin
+will now wait until a `conf` file exists before appending its configuration.
+Additionally, this change includes a bug fix for topology aware service
+routing.
+
+* Changed CNI plugin installer to always run in 'chained' mode; the plugin will
+  now wait until another CNI plugin is installed before appending its
+  configuration
+* Added a timeout value to the HttpRoute CRD. The field is currently unused but
```

Comment:
```suggestion
* Added a `timeouts` field to `rule`s in the HTTPRoute CRD. The field is currently unused, but
```

Reply:
also, i'm somewhat uncertain if this is a user-facing change, since...it doesn't *currently* do anything. it might be better to only add it to the changelog when it's actually fully implemented? I dunno.

Reply:
Hm yeah, I'm leaning towards adding it in when it's fully implemented. We did bump up the chart version though 🤔 think it should be fine not to mention... Would rather just announce it all at once.

Reply:
Going to remove it.